### ISSUE TITLE
Errors from multiple functions

### ISSUE THREAD

**Issue by Zargith** [Androz2091/discord-player#175]
Hello, I'm upgrading my bot's music system using this lib and during my tests I encountered some errors from the lib.

Command skip, stop pause, resume and setVolume. In order:
```
TypeError: Cannot read property 'end' of null
    at Player.stop (/my/project/path/node_modules/discord-player/src/Player.js:398:42)
```
```
TypeError: Cannot read property 'end' of null
    at Player.stop (/my/project/path/node_modules/discord-player/src/Player.js:453:42)
```
```
TypeError: Cannot read property 'pause' of null
    at Player.stop (/my/project/path/node_modules/discord-player/src/Player.js:365:42)
```
```
TypeError: Cannot read property 'resume' of null
    at Player.stop (/my/project/path/node_modules/discord-player/src/Player.js:380:42)
```
```
TypeError: Cannot read property 'setVolumeLogarithmic' of null
    at Player.stop (/my/project/path/node_modules/discord-player/src/Player.js:416:42)
```

I got an error when my queue ended and I tried to add another song. The only log that I have (and don't really understand):
```
events.js:291
      throw er; // Unhandled 'error' event
      ^

Error: Could not find player config
    at exports.getBasicInfo (/my/project/path/node_modules/ytdl-core/lib/info.js:59:13)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async Object.exports.<computed> [as getBasicInfo] (/my/project/path/node_modules/ytdl-core/lib/info.js:296:18)
    at async exports.getFullInfo (/my/project/path/node_modules/ytdl-core/lib/info.js:187:14)
    at async Object.exports.<computed> [as getFullInfo] (/my/project/path/node_modules/ytdl-core/lib/info.js:296:18)
Emitted 'error' event on Encoder instance at:
    at emitErrorNT (internal/streams/destroy.js:100:8)
    at emitErrorCloseNT (internal/streams/destroy.js:68:3)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
```

I also have a bug (I guess) when I add a music, sometimes it tells me:
> undefined has been added to the queue!

Which is from
```
client.player.on('trackAdd', (message, track) => message.channel.send(`${track.title} has been added to the queue!`));
```
Versions:
nodejs: 14.8.0
npm: 6.14.8
discord.js: 12.5.1
discord-player: 3.0.3

**Comment by ThaddeusKKR** [comment]
Same issue with skip commands and others

**Comment by hands8142** [comment]
fixed undefined #180

**Comment by Zargith** [comment]
Well it appears I still got the problem even with the last version (3.2.1) :

```
TypeError: Cannot read property 'setVolumeLogarithmic' of null
    at Player.setVolume (/my/project/path/node_modules/node_modules/discord-player/src/Player.js:441:42)
```

```
TypeError: Cannot read property 'end' of null
    at Player.stop (/my/project/path/node_modules/discord-player/src/Player.js:423:42)
```
